src/App/Expression.cpp

```diff
@@ -1728,7 +1728,7 @@ FunctionExpression::FunctionExpression(const DocumentObject *_owner, Function _f
     : UnitExpression(_owner)
     , f(_f)
     , fname(std::move(name))
-    , args(_args)
+    , args(std::move(_args))
```

Comment:
Can't we get the same effect by changing it to a reference in the call and let the conversation here to a non-reference handle the copy?

I guess I just don't like std::move if there's an alternative 😅

Reply:
We could, although I was under the impression that modern usage was to go the other way, _a la_ the "pass-by-value and move idiom" so you're handling both lvalues and rvalues as efficiently as possible.

Reply:
I've missed that memo, but looked it up and I see the point. Feel free to resolve this :)